evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -91,10 +91,12 @@ def validate(self, X, y):
             error_msg = "The number of instances of these targets is less than 2 * the number of cross folds = {} instances: {}"
             DataCheck._add_message(
                 DataCheckError(
-                    message=error_msg.format(self.cv_folds, below_threshold_values),
+                    message=error_msg.format(
```

Comment:
Something about the new pandas caused the order in the datachecks details to change.  I sorted the ones that got changed so it shouldn't be an issue going forward.

Reply:
Must be `value_counts`? It's called in both this data check and InvalidTargetDataCheck

Reply:
Frustrating, I believe they changed the behavior of `value_counts` not too many releases ago too 🤔